### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by whyboris**
It's a rainy day and I'm not running around Tokyo exploring -- had enough time to create a minor PR 😅 

I might find time again in the future to do more VHA work, though I won't be back from Tokyo until end of June 🗼 

This PR adds minor fixes along with:
- folder styling (feedback welcome - don't mean to step too much on your work 🙇 )

Cheers 🙌

### CODE REVIEW

File: src/app/components/home/clip/clip.component.ts [hunk 1]
```diff
@@ -27,14 +27,13 @@ export class ClipComponent implements OnInit {
   @Input() largerFont: boolean;
   @Input() showMeta: boolean;
 
+  // numberOfThumbs = 0;
```
Comment (whyboris) [inline src/app/components/home/clip/clip.component.ts]: This variable is never used so I commented it out; I suspect I can delete it safely -- or did you have some use for it in mind in the future? @cal2195

Comment (cal2195) [reply src/app/components/home/clip/clip.component.ts]: Yeah, I think I was just duplicating other code and forgot about this! 👍

**Comment by cal2195** [comment]
Just saw the "up button" - so sweet! 😄

**Comment by whyboris** [comment]
Thanks for the feedback on the folder icon -- I think I made it better (with respect to both of our preferences). 

I was a bit lazy, so I merged in the change into #174 with https://github.com/whyboris/Video-Hub-App/pull/174/commits/ce00d9452a731406064d55f48a931f3edbdd133c

I'll merge this in and the other branch will patch it up 👍

**Comment by whyboris** [comment]
Also removed commented-out code 👍 with https://github.com/whyboris/Video-Hub-App/pull/174/commits/94286cbe2ac6d8236de01ef0670bb69dd782dd2e

**Pull request merged**